fix: issues with kcat guide (#485)
redhat-developer/app-services-guides: docs/kafka/kcat-kafka/README.adoc

NOTE: {product-kafka} also supports the SASL/OAUTHBEARER mechanism for authentication, which is the recommended authentication mechanism to use. However, Kafkacat does not yet fully support OAUTHBEARER, so this example uses SASL/PLAIN.

> NOTE: kafkacat producer might behave differently depending on the operating system. For Mac operating systems 
> messages can be sent using pipe. For example "echo "message" | kcat ... " 

Author: ```suggestion
NOTE: kafkacat producer might behave differently depending on the operating system. For Mac operating systems messages can be sent by redirecting them to the kafkacat binary. For example "echo "message" | kafkacat ... " 
```

NOTE: {product-kafka} also supports the SASL/OAUTHBEARER mechanism for authentication, which is the recommended authentication mechanism to use. However, Kafkacat does not yet fully support OAUTHBEARER, so this example uses SASL/PLAIN.

> NOTE: kafkacat producer might behave differently depending on the operating system. For Mac operating systems messages can be sent by redirecting them to the kafkacat binary. For example "echo "message" | kafkacat ... " 

Reviewer: ```suggestion
NOTE: A Kafkacat producer might behave differently depending on the operating system. For example, on Mac operating systems, messages are sent by redirecting them to the Kafkacat binary in the format `echo "message" | kafkacat ...`.
```
